### PR TITLE
Fix area calculation by using correct base area

### PR DESCRIPTION
**Pull request by MichaMans**
Thanks to @marcusfuchs this implements the correct area calculation, see #662 #663

### CODE REVIEW

**Comment by MichaMans** [comment]
@PRemmen unfortunately I needed to transfer this to an internal branch. I updated the version number. I would merge this after you approved, checks passing, seems a good solution

**Review by PRemmen — APPROVED** [review]

**Comment by MichaMans** [comment]
this closes #662

**Pull request merged**
